### PR TITLE
Dynamic footer buttons for Insert Slide Modal (#665)

### PR DESCRIPTION
**Pull request by AkalUstat**
Depending on what 'page' is currently open, the footer button for changing 'pages' changes.
*Dhan Guru____ slides age:*
Screen Shot 2019-07-08 at 12 54 08 PM
(Go to announcements page)
*Announcements Page:*Screen Shot 2019-07-08 at 12 54 12 PM
(Go back to dhan guru___ slides page)

### CODE REVIEW

File: www/js/insert-slide.js [hunk 1]
```diff
@@ -153,24 +140,30 @@ modal.addFooterBtn('Close', 'tingle-btn tingle-btn--pull-right tingle-btn--dange
...
+  'Announcement',
+  'tingle-btn tingle-btn-pull-left tingle-btn--default modal-tab-btn',
+  () => {},
+);
+const modalTabBtn = document.querySelector('.modal-tab-btn');
+modalTabBtn.addEventListener('click', () => {
+  if (!isAnnouncementTab) {
+    isAnnouncementTab = true;
+    // sets up modal content (box, etc)
+    modal.setContent(announcementPage);
```
Comment (tsingh777) [inline www/js/insert-slide.js]: Can you make the 2 contents the same height? that way the element doesn't bounce.

Comment (AkalUstat) [reply www/js/insert-slide.js]: There would be a ton of whitespace/unused space

Comment (tsingh777) [reply www/js/insert-slide.js]: make the announcement box bigger.

File: www/js/insert-slide.js [hunk 2]
```diff
@@ -153,24 +140,30 @@ modal.addFooterBtn('Close', 'tingle-btn tingle-btn--pull-right tingle-btn--dange
...
+    setLangSliderVal();
+    boxInputFunctionality();
+    modalTabBtn.textContent = 'Back';
```
[the remark this answers is not in the document]
Comment (maneetpaul) [reply www/js/insert-slide.js]: I think it does. It only shows the back button when you are in the announcements section. So when you press back, it brings you back to the main Dhan Guru selection screen

[the remark this answers is not in the document]
Comment (tsingh777) [reply www/js/insert-slide.js]: but if you go to announcements and then close and reopen the modal, you are still on announcements and it shows back. that could get confusing imo. if you're fine with it then we can see if we can get any feedback for it.

[the remark this answers is not in the document]
Comment (AkalUstat) [reply www/js/insert-slide.js]: I could make it change back on close

[the remark this answers is not in the document]
Comment (maneetpaul) [reply www/js/insert-slide.js]: yes please, it should open to the Dhan Guru selector whenever the modal is opened.

**Comment by tsingh777** [comment]
The button still says back when reopening after switching to announcement event though the content is the dhan guru slides.

**Pull request merged**